logbook-httpclient/src/main/java/org/zalando/logbook/httpclient/LocalRequest.java

```diff
@@ -1,16 +1,18 @@
 package org.zalando.logbook.httpclient;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+import static java.util.stream.Collectors.*;
+import static org.zalando.fauxpas.FauxPas.throwingUnaryOperator;
```

Comment:
Could you please make sure that your editor does not change the existing code formatting? Within Logbook, the `import static` lines are always after the normal `import`s.